data/types/bolus/iob/iob_test.go

```diff
@@ -0,0 +1,105 @@
+package iob_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+
+	dataNormalizer "github.com/tidepool-org/platform/data/normalizer"
+	"github.com/tidepool-org/platform/data/types/bolus/iob"
+	dataTypesTest "github.com/tidepool-org/platform/data/types/test"
+	errorsTest "github.com/tidepool-org/platform/errors/test"
+	"github.com/tidepool-org/platform/pointer"
+	"github.com/tidepool-org/platform/structure"
+	structureValidator "github.com/tidepool-org/platform/structure/validator"
+	"github.com/tidepool-org/platform/test"
+)
+
+func NewIob() *iob.Iob {
+	datum := iob.NewIob()
+	datum.InsulinOnBoard = pointer.FromFloat64(test.RandomFloat64FromRange(iob.InsulinOnBoardMinimum, iob.InsulinOnBoardMaximum))
+	return datum
+}
+
+func CloneIob(datum *iob.Iob) *iob.Iob {
+	if datum == nil {
+		return nil
+	}
+	clone := iob.NewIob()
+	clone.InsulinOnBoard = pointer.CloneFloat64(datum.InsulinOnBoard)
+	return clone
+}
```

Comment:
Functions already in  data/types/bolus/iob/test/iob.go

Reply:
yep, let's leverage the ones defined in test